libmamba/src/core/shell_init.cpp

```diff
@@ -515,7 +516,7 @@ namespace mamba
     if $env.CONDA_PROMPT_MODIFIER? != null {
       # unset set variables
       for x in (^$env.MAMBA_EXE shell deactivate --shell nu
-              | split row ";") {
+              | split row (if $nu.os-info.name == "windows" { ";" } else { ";" }) {
```

Comment:
Should it be `(if $nu.os-info.name == "windows" { ";" } else { ":" })`?

Reply:
yah, but the else part, it should be `:`, I guess line 519 should be the same as line 500

Reply:
Ah right you are correct ;-) Fixed!